airflow/providers/cncf/kubernetes/utils/xcom_sidecar.py

```diff
@@ -47,13 +47,15 @@ class PodDefaults:
     )
 
 
-def add_xcom_sidecar(pod: k8s.V1Pod) -> k8s.V1Pod:
+def add_xcom_sidecar(pod: k8s.V1Pod, **kwargs) -> k8s.V1Pod:
```

Comment:
when i said kwargs only i meant this
```suggestion
def add_xcom_sidecar(pod: k8s.V1Pod, *, sidecar_container_image=None) -> k8s.V1Pod:
```

the `*` makes everything after kwargs only, so that positional doesn't work